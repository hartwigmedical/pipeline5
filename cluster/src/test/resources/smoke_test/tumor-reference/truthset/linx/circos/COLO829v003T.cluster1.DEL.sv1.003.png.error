Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 12.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 13.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 44.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value $track_type in hash element at /opt/tools/circos/0.69.6/bin/../lib/Circos.pm line 2402.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
