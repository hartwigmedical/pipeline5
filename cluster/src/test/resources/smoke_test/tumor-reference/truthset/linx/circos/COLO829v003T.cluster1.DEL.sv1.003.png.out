debuggroup summary 0.23s welcome to circos v0.69-6 31 July 2017 on Perl 5.024001
debuggroup summary 0.23s current working directory /
debuggroup summary 0.23s command /opt/tools/circos/0.69.6/bin/circos -nosvg -conf /data/output/circos/COLO829v003T.cluster1.DEL.sv1.circos.003.conf -outputdir /data/output/plot -outputfile COLO829v003T.cluster1.DEL.sv1.003.png
debuggroup summary 0.23s loading configuration from file /data/output/circos/COLO829v003T.cluster1.DEL.sv1.circos.003.conf
debuggroup summary 0.23s found conf file /data/output/circos/COLO829v003T.cluster1.DEL.sv1.circos.003.conf
debuggroup summary 0.39s debug will appear for these features: output,summary
debuggroup summary 0.39s bitmap output image /data/output/plot/COLO829v003T.cluster1.DEL.sv1.003.png
debuggroup summary 0.39s parsing karyotype and organizing ideograms
debuggroup summary 0.39s karyotype has 1 chromosomes of total size 974
debuggroup summary 0.40s applying global and local scaling
debuggroup summary 0.40s allocating image, colors and brushes
debuggroup summary 7.62s drawing 1 ideograms of total size 974
debuggroup summary 7.62s drawing highlights and ideograms
debuggroup summary 7.62s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.bg.conf
debuggroup summary 7.62s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.bg.conf
debuggroup summary 7.63s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.bg.conf
debuggroup summary 7.63s process track_0 highlight /data/output/circos/COLO829v003T.cluster1.DEL.sv1.gene.circos
debuggroup summary 7.63s process track_1 highlight /data/output/circos/COLO829v003T.cluster1.DEL.sv1.exon.circos
debuggroup summary 7.63s process track_2 highlight /data/output/circos/COLO829v003T.cluster1.DEL.sv1.segment.circos
debuggroup summary 7.71s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/link.conf
debuggroup summary 7.71s process track_0 link /data/output/circos/COLO829v003T.cluster1.DEL.sv1.link.circos
debuggroup summary 7.71s drawing link track_0 z 0
WARNING *** Track id [track_3] has no type definition. This is alright if you're using the track for side-effects like axes and backgrounds.
WARNING *** Track id [track_3] of type [_undef_] has no file definition. This is alright if you're using the track for side-effects like axes and backgrounds.
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/text.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/text.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/text.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/text.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/connector.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.conf
debuggroup summary 7.72s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.conf
debuggroup summary 7.72s processing track_0 text /data/output/circos/COLO829v003T.cluster1.DEL.sv1.position.circos
debuggroup summary 7.73s processing track_1 text /data/output/circos/COLO829v003T.cluster1.DEL.sv1.exon.rank.circos
debuggroup summary 7.73s processing track_2 text /data/output/circos/COLO829v003T.cluster1.DEL.sv1.gene.name.circos
debuggroup summary 7.73s processing track_4 histogram /data/output/circos/COLO829v003T.cluster1.DEL.sv1.cna.circos
debuggroup summary 7.73s processing track_5 text /data/output/circos/COLO829v003T.cluster1.DEL.sv1.distance.circos
debuggroup summary 7.73s processing track_6 histogram /data/output/circos/COLO829v003T.cluster1.DEL.sv1.cna.circos
debuggroup summary 7.73s processing track_7 histogram /data/output/circos/COLO829v003T.cluster1.DEL.sv1.map.circos
debuggroup summary 7.73s processing track_8 histogram /data/output/circos/COLO829v003T.cluster1.DEL.sv1.map.circos
debuggroup summary 7.73s processing track_9 connector /data/output/circos/COLO829v003T.cluster1.DEL.sv1.connector.circos
debuggroup summary 7.73s processing track_10 scatter /data/output/circos/COLO829v003T.cluster1.DEL.sv1.scatter.circos
debuggroup summary 7.74s processing track_11 scatter /data/output/circos/COLO829v003T.cluster1.DEL.sv1.scatter.sgl.circos
debuggroup summary 7.74s processing track_12 highlight /data/output/circos/COLO829v003T.cluster1.DEL.sv1.line_element.circos
debuggroup summary 7.74s processing track_13 highlight /data/output/circos/COLO829v003T.cluster1.DEL.sv1.fragile.circos
debuggroup summary 7.74s drawing track_0 text z 0 COLO829v003T.cluster1.DEL.sv1.position.circos 
debuggroup summary 7.74s placing text track COLO829v003T.cluster1.DEL.sv1.position.circos
debuggroup summary 7.74s ... see progress with -debug_group text
debuggroup summary 7.74s ... see placement summary with -debug_group textplace
debuggroup summary 7.82s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 7.82s drawing track_1 text z 0 COLO829v003T.cluster1.DEL.sv1.exon.rank.circos 
debuggroup summary 7.82s placing text track COLO829v003T.cluster1.DEL.sv1.exon.rank.circos
debuggroup summary 7.82s ... see progress with -debug_group text
debuggroup summary 7.82s ... see placement summary with -debug_group textplace
debuggroup summary 7.86s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 7.87s drawing track_2 text z 0 COLO829v003T.cluster1.DEL.sv1.gene.name.circos 
debuggroup summary 7.87s placing text track COLO829v003T.cluster1.DEL.sv1.gene.name.circos
debuggroup summary 7.87s ... see progress with -debug_group text
debuggroup summary 7.87s ... see placement summary with -debug_group textplace
debuggroup summary 7.91s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 7.92s drawing track_3 _undef_ z 0 no_file 
debuggroup summary 7.92s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 7.97s drawing track_4 histogram z 0 COLO829v003T.cluster1.DEL.sv1.cna.circos orient out
debuggroup summary 7.97s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.00s drawing track_5 text z 0 COLO829v003T.cluster1.DEL.sv1.distance.circos 
debuggroup summary 8.00s placing text track COLO829v003T.cluster1.DEL.sv1.distance.circos
debuggroup summary 8.00s ... see progress with -debug_group text
debuggroup summary 8.00s ... see placement summary with -debug_group textplace
debuggroup summary 8.04s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.04s drawing track_6 histogram z 0 COLO829v003T.cluster1.DEL.sv1.cna.circos orient out
debuggroup summary 8.04s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.07s drawing track_7 histogram z 0 COLO829v003T.cluster1.DEL.sv1.map.circos orient out
debuggroup summary 8.07s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.10s drawing track_8 histogram z 0 COLO829v003T.cluster1.DEL.sv1.map.circos orient out
debuggroup summary 8.10s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.12s drawing track_9 connector z 0 COLO829v003T.cluster1.DEL.sv1.connector.circos 
debuggroup summary 8.12s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.13s drawing track_10 scatter z 0 COLO829v003T.cluster1.DEL.sv1.scatter.circos orient out
debuggroup summary 8.13s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.13s drawing track_11 scatter z 0 COLO829v003T.cluster1.DEL.sv1.scatter.sgl.circos orient out
debuggroup summary 8.13s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.13s drawing track_12 highlight z 0 COLO829v003T.cluster1.DEL.sv1.line_element.circos 
debuggroup summary 8.13s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 8.13s drawing track_13 highlight z 0 COLO829v003T.cluster1.DEL.sv1.fragile.circos 
debuggroup summary 8.13s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup output 8.13s generating output
debuggroup output 8.61s created PNG image /data/output/plot/COLO829v003T.cluster1.DEL.sv1.003.png (313 kb)
